tests/failover_test.py

```diff
@@ -34,6 +34,7 @@
 _KubernetesServerRunner = k8s_xds_server_runner.KubernetesServerRunner
 
 
+@absltest.skip("switching to a single regional sandbox due to b/397859175")
```

Comment:
I think we should use existing framework function `is_supported` to skip this test:
https://github.com/grpc/psm-interop/blob/b753b6827d9cf9bef4fca0f68365f54c1af5a56c/tests/app_net_csm_observability_test.py#L176